scripts/check_unstaged.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$FILES" != "" ]]; then
 	log "These are the changes:"
 	log
 	for file in "${files[@]}"; do
-		git --no-pager diff "$file" 1>&2
+		git --no-pager diff -- "$file" 1>&2
```

Comment:
related?

Reply:
Yep, could technically be a separate change, but some filenames break this script.